sdk/lib/3rdparty/freetype/src/psaux/psintrp.c

```diff
@@ -2613,19 +2618,13 @@
            *
            */
 #ifdef __REACTOS__
-          CF2_HintMapRec *counterHintMap = malloc(sizeof(CF2_HintMapRec));
-          CF2_HintMaskRec  counterMask;
-          if (!counterHintMap)
-          {
-            lastError = FT_Err_Out_Of_Memory;
-            goto exit;
-          }
-/* Ugly but it allows us to reduce the diff */
-#define counterHintMap (*counterHintMap)
+          CF2_HintMapRec  *counterHintMap = malloc(sizeof(*counterHintMap));
```

Comment:
Please restore the NULL-check + lastError and goto.

Reply:
Restored.